src/Security/Authentication/BearerToken/src/BearerTokenOptions.cs

```diff
@@ -23,14 +23,26 @@ public BearerTokenOptions()
     /// The expiration information is stored in the protected token. Because of that, an expired token will be rejected
     /// even if it is passed to the server after the client should have purged it.
     /// </summary>
+    /// <remarks>
+    /// Defaults to 1 hour.
+    /// </remarks>
     public TimeSpan BearerTokenExpiration { get; set; } = TimeSpan.FromHours(1);
 
     /// <summary>
-    /// If set, the <see cref="BearerTokenProtector"/> is used to protect and unprotect the identity and other properties which are stored in the
-    /// bearer token value. If not provided, one will be created using <see cref="TicketDataFormat"/> and the <see cref="IDataProtectionProvider"/>
+    /// Controls how much time the refresh token will remain valid from the point it is created.
+    /// The expiration information is stored in the protected token.
+    /// </summary>
+    /// <remarks>
+    /// Defaults to 14 days.
+    /// </remarks>
+    public TimeSpan RefreshTokenExpiration { get; set; } = TimeSpan.FromDays(14);
```

Comment:
just curious, are these defaults we are making up? or there's some RFC or something we should ref?

Reply:
I didn't see any suggested default for the refresh token lifetime in the oauth2 RFC [1] or any suggestion that it necessarily needs to expire at all. 

Microsoft identity platform default to 24 hours for SPAs and 90 days for everything else [2]. Auth0 defaults to 30 days [3]. And our Cookie auth handler's cookie expiration defaults to 14 days [4].

Since cookies have sliding expiration similar to refresh by default, and cookies and refresh tokens are created the same way by default, I went with the Cookie auth handler's cookie expiration of 14 days.

1. https://datatracker.ietf.org/doc/html/rfc6749#section-1.5
2. https://learn.microsoft.com/en-us/azure/active-directory/develop/refresh-tokens#token-lifetime
3. https://auth0.com/docs/secure/tokens/refresh-tokens/configure-refresh-token-expiration
4. https://github.com/dotnet/aspnetcore/blob/e4c6621d9a09cae8565ad918ba5083c9b28840bd/src/Security/Authentication/Cookies/src/CookieAuthenticationOptions.cs#L30